dags/glam_dev.py

```diff
@@ -0,0 +1,444 @@
+"""
+DEV Desktop ETL for https://glam.telemetry.mozilla.org/
+
+Generates and runs a series of BQ queries, see
+[bigquery_etl/glam](https://github.com/mozilla/bigquery-etl/tree/main/bigquery_etl/glam)
+in bigquery-etl and the
+[glam_subdags](https://github.com/mozilla/telemetry-airflow/tree/main/dags/glam_subdags)
+in telemetry-airflow.
```

Comment:
I'd add a note to future Airflow triagers on what to expect in terms of stability or (not) do in case of failures.